Tests/MountebankSwiftTests/ExampleData/UIImage+Example.swift

```diff
@@ -25,7 +25,7 @@ struct StubImage {
 // swiftlint:disable force_unwrapping
 
 extension StubImage {
-    static let image = StubImage(data: StubImage.example.value)!
+    @MainActor static let image = StubImage(data: StubImage.example.value)!
```

Comment:
This line can be removed. If you want to get rid of this whole file you can also inline this:

```
        fileprivate static let base64Image = "iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAAXNSR0IArs4c6QAAAE" +
        "RlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAABaADAAQAAAABAAAABQAAAAB" +
        "/qhzxAAAAKUlEQVQIHWP8//8/AwMjI5CAgv//wTyEAFScCaYAmcYhCDQDWRUDkA8AEGsMAtJaFngAAAAASUVORK5CYII="

        static let binary = Example(
            value: Is(
                statusCode: 200,
                body: Data(base64Encoded: base64Image, options: .ignoreUnknownCharacters)!
            ),
            json: [
                "_mode" : "binary",
                "statusCode" : 200,
                "body" : .string(base64Image),
            ]
        )

```

in `Is+Examples.swift`